modules/spring-config/src/main/java/org/mule/runtime/config/internal/dsl/model/extension/xml/MacroExpansionModuleModel.java

```diff
@@ -125,7 +125,7 @@ public class MacroExpansionModuleModel {
    * If true avoid the creation of an implicit configuration
    */
   private final boolean disable_xml_sdk_implicit_configuration_creation =
-      valueOf(getProperty(MULE_DISABLE_XML_SDK_IMPLICIT_CONFIGURATION_CREATION, "false"));
+      valueOf(getProperty(MULE_DISABLE_XML_SDK_IMPLICIT_CONFIGURATION_CREATION, "true"));
```

Comment:
Would this only be done en 4.4.x? What is the behavior in 4.3.x? Will master still default to false?

Reply:
Yes, won't be on 4.3 and 4.5. Someone should fix this problem in the next release and set the property to false again

Reply:
I think that the issue with this would be that we would be breaking compatibility for someone who is using implicit configs without setting the property.I will talk with the team to see if we can take the issue and fix it before the next cutoff.